### DISCUSSION TITLE
is there a setting to delay the schedule when the pc is powered off?

### DISCUSSION THREAD

**Discussion by StefB3n** [creativeprojects/resticprofile#252] [Q&A] [answered]
i have been experimenting with resticprofile scheduling, and i was wondering if there was a setting that makes the scheduled backup start at the specified hour and if the pc is off it waits for it to come on. right now it backups only if the pc is on at the specified time

i know it's possible on windows task scheduler but i wanted it in resticprofile
is there a setting to do it?

(if needed, my profile)
{
    "version": "1",
    "default": {
      "repository": "rest:*****",
      "password-file": "word.txt",
      "exclude": "exc.txt",
      "backup": {
        "schedule": "9:00",
        "schedule-permission": "user",
        "schedule-priority": "background",
        "schedule-log": "profile-backup.log",
        "schedule-lock-mode": "default",
        "schedule-lock-wait": "1h15m30s",
        "verbose": true,
        "source": [
          "*******"
        ]
      }
    }
  }

**Comment by creativeprojects** [comment, marked as answer]
Hi,
No there's no setting to do that right now.
As a workaround you can probably update the task scheduler manually. I know it's not very convenient but that should work.

I'll have a look if the other scheduling could provide the same functionality, in which case I can add a flag for it.

**Reply by StefB3n** [reply]
ok, thanks
